web/client/components/misc/datetimepicker/DateTimePicker.js

```diff
@@ -152,7 +153,14 @@ class DateTimePicker extends Component {
                 </div>
                 <div className={`rw-calendar-popup rw-popup-container ${popupPosition === 'top' ? 'rw-dropup' : ''} ${!calendarVisible ? 'rw-popup-animating' : ''}`} style={{ display: calendarVisible ? 'block' : 'none', overflow: calendarVisible ? 'visible' : 'hidden', height: '285px' }}>
                     <div className={`rw-popup`} style={{ transform: calendarVisible ? 'translateY(0)' : 'translateY(-100%)', padding: '0', borderRadius: '4px', position: calendarVisible ? '' : 'absolute' }}>
-                        <Calendar tabIndex="-1" ref={this.attachCalRef} onMouseDown={this.handleMouseDown} onChange={this.handleCalendarChange} {...props} value={new Date(this.props.value)}/>
+                        <Calendar tabIndex="-1"
+                            ref={this.attachCalRef}
+                            onMouseDown={this.handleMouseDown}
+                            onChange={this.handleCalendarChange}
+                            {...props}
+                            value={!isNil(this.props.value) ? new Date(this.props.value) : undefined}
+                            currentDate={!isNil(this.props.value) ? new Date(this.props.value) : new Date() }
```

Comment:
this was missing before, can you point here in the comment a reference to the doc ?

Reply:
Before the value was fixed manually initial value, set to today by default. This was incorrect conceptually (the value of the `DateTimePicker` was internally set to today, but not displaying it, and it was causing wrong value to be assigned and inconsistency in state, causing various issues.
I fixed this in this PR https://github.com/geosolutions-it/MapStore2/pull/8850

But then when you open the calendar it was starting from 1970. This PR simply correct this behavior of the calendar, setting the proper "currentDate"  (position where to open the calendar) to today if the value is undefined, opening the calendar at today instead of Jan 1970 (implicit conversion 0 / undefined value).